bin/pull_request_blaster_outer.rb

```diff
@@ -7,29 +7,18 @@
 require 'trollop'
 
 opts = Trollop.options do
-  opt :repo,    "The repo to update. If not passed, will try all repos in config/repos.yml. For example: --repo manageiq or --repo miq-test/sandbox", :type => :strings
-  opt :base,    "The name of the branch you want the changes pulled into.",                              :type => :string, :required => true
-  opt :head,    "The name of the branch containing the changes.",                                        :type => :string, :required => true
-  opt :script,  "The path to the script that will update the desired files. See the scripts directory.", :type => :string, :required => true
-  opt :message, "The commit message and PR title for this change.",                          :type => :string, :required => true
-  opt :dry_run, "Make local changes, but don't fork, push, or create the pull request.",     :default => false
-end
+  opt :base,    "The name of the branch you want the changes pulled into.",   :type => :string, :required => true
+  opt :head,    "The name of the branch containing the changes.",             :type => :string, :required => true
+  opt :script,  "The path to the script that will update the desired files.", :type => :string, :required => true
+  opt :message, "The commit message and PR title for this change.",           :type => :string, :required => true
 
-if opts[:repo]
-  repos = opts[:repo].collect do |repo_opt|
-    org, repo = repo_opt.split("/")
-    org, repo = "ManageIQ", org if repo.nil?
-    ManageIQ::Release::Repo.new(repo, :org => org)
-  end
-else
-  repos = ManageIQ::Release::Repos["master"]
+  opt :repo,    "The repo to update. If not passed, will try all repos in config/repos.yml.", :type => :strings
+  opt :dry_run, "Make local changes, but don't fork, push, or create the pull request.", :default => false
 end
 
 results = {}
-repos.each do |repo|
-  puts ManageIQ::Release.header(repo.github_repo)
+ManageIQ::Release.repos_for(opts[:repo]).each do |repo|
```

Comment:
Should this be `each_repo(opts[:repo])` instead?

Reply:
Yes...good catch.